package.json

```diff
@@ -4,6 +4,8 @@
   "private": true,
   "dependencies": {
     "@material-ui/core": "^4.12.3",
+    "@material-ui/icons": "^4.11.2",
```

Comment:
We could consider using MUI v5 beta, it should move to stable in a couple of months tops.